File: sdkwork-app-sdk-typescript/src/types/vip-privilege-usage-vo.ts
/** VIP特权使用情况响应 */
export interface VipPrivilegeUsageVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 加速特权已使用次数 */
  speedUpUsed?: number;
  /** 加速特权限制次数 */
  speedUpLimit?: number;
  /** 优先队列已使用次数 */
  priorityQueueUsed?: number;
  /** 优先队列限制次数 */
  priorityQueueLimit?: number;
  /** 专属模型已使用次数 */
  exclusiveModelUsed?: number;
  /** 专属模型限制次数 */
  exclusiveModelLimit?: number;
}

File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/vip_privilege_usage_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class VipPrivilegeUsageVO:
    """VIP特权使用情况响应"""
    created_at: str = None
    updated_at: str = None
    speed_up_used: int = None
    speed_up_limit: int = None
    priority_queue_used: int = None
    priority_queue_limit: int = None
    exclusive_model_used: int = None
    exclusive_model_limit: int = None
